### PR TITLE
Auth Page

### PR DESCRIPTION
**Pull request by mahmood100100**

### CODE REVIEW

File: src/layouts/UserLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Navigate } from 'react-router-dom';
+function UserLayout(){
+  const token: string | null = localStorage.getItem('user_profile_token');
+  
+  if (!token) {
+    return <Navigate to="/auth" replace />;
```
Comment (IbrahimLama) [inline src/layouts/UserLayout.tsx:7]: token will be needed to be saved in the redux store as it will be used in multiple components

Comment (mahmood100100) [reply src/layouts/UserLayout.tsx:7]: yes, of course. I just used it here initially and will move it to the redux store.

File: src/components/signIn/SignIn.tsx [hunk 2]
```diff
@@ -1,42 +1,83 @@
...
+  });
+
+
   return (
-    <div className="a-right">
-        <form className="infoForm authForm">
-          <h3>Log In</h3>
-  
-          <div>
+    <div className={styles['right-section']}>
```
Comment (IbrahimLama) [inline src/components/signIn/SignIn.tsx:31]: don't use - in classnames to e able to use the dot property accessor way. styles.rightSection

Comment (mahmood100100) [reply src/components/signIn/SignIn.tsx:31]: ok

**Comment by mahmood100100** [comment]
the branch solution has been renamed to Auth

**Pull request merged**